test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -1732,12 +1753,13 @@ def forward(self, x: int, y: int):
         y = 2
         self.run_test(ArithmeticModule(), (x, y))
 
-    @disableScriptTest()
+    # In tracing, None outputs are removed. In scripting they're kept but
```

Comment:
does this mean we can enable test for optional outputs? e.g. None in one branch, some tensor in other.

Reply:
Yes. Tests for that are in test/onnx/test_pytorch_onnx_no_runtime.py